specs/glossary.md

```diff
@@ -186,57 +187,76 @@ cf. [Execution Engine Specification](exec-engine.md)
 Payload attributes were historically called "L2 block inputs" in the L2 spec and you might still hear some people using
 this term.
 
-## L1 Attributes Transaction
+## Deposits
 
-[l1-attributes-tx]: /glossary.md#l1-attributes-transaction
+[deposits]: glossary.md#deposits
 
-A transaction with an Optimism-specific transaction type, that is used to register the L1 block attributes
-(number, timestamp, ...) on L2.
+A deposit is an L2 transaction derived from an L1 block.
 
-The L1 attributes for a given L1 block can be read on L2 from the [L1 Attributes Predeployed
-Contract][l1-attr-predeploy].
+There are two kinds of deposits:
 
-cf. [L1 attributes transaction format](/rollup-node.md#payload-transaction-format) (in the section on [payload
-attributes])
+- [L1 Attributes Deposit][l1-attribute-tx], which submit the L1 block's attributes to the [L1 Attributes Predeployed
```

Comment:
Github doesn't recognize the first pair of square brackets as a link

Reply:
Yup, that should be `[l1-attributes-deposit]`.